File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/marketingvb/vb/distributecampaignfrommarketinglist7.vb
' <snippetdistributecampaignfrommarketinglist7>


          ' Copy the marketing list.  First create a new one, and then copy over the
	' members.
	list.ListName = list.ListName &amp; " Copy"
	_copiedMarketingListId = _serviceProxy.Create(list)
          Dim copyRequest = New CopyMembersListRequest With
                            {
                                .SourceListId = _marketingListId,
                                .TargetListId = _copiedMarketingListId
                            }

	_serviceProxy.Execute(copyRequest)

' </snippetdistributecampaignfrommarketinglist7>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/marketing/cs/distributecampaignfrommarketinglist7.cs
// <snippetdistributecampaignfrommarketinglist7>


 // Copy the marketing list.  First create a new one, and then copy over the
 // members.
 list.ListName = list.ListName + " Copy";
 _copiedMarketingListId = _serviceProxy.Create(list);
 var copyRequest = new CopyMembersListRequest
 {
     SourceListId = _marketingListId,
     TargetListId = _copiedMarketingListId
 };

 _serviceProxy.Execute(copyRequest);

// </snippetdistributecampaignfrommarketinglist7>